docs/Checksum_Deprecation.md

```diff
@@ -1,22 +1,19 @@
 # MD5 and SHA-1 Deprecation
 
-During early 2015 Homebrew started the process of deprecating _SHA1_ for package
+In early 2015 Homebrew started the process of deprecating _SHA1_ for package
 integrity verification. Since then every formulae under the Homebrew organisation
```

Comment:
`every formula` not `every formulae`

Reply:
We could even just use `formulae` really, since it is a plural.

Reply:
yeah anything that is actually … English … would be better

Reply:
I think in the classic case of open source documentation I threw this together in a hurry & it has barely been touched since 😅.